rcl_lifecycle/CMakeLists.txt

```diff
@@ -104,6 +104,7 @@ if(BUILD_TESTING)
   endif()
   ament_add_gtest(test_rcl_lifecycle
     test/test_rcl_lifecycle.cpp
+    TIMEOUT 400
```

Comment:
400 secs is the longest timeout for the test in `rcl`... 

with my local environment with `rmw_fastrtps_cpp`, it can complete the test in 3 seconds.

```bash
root@tomoyafujita:~/ros2_ws/colcon_ws# colcon test --event-handlers console_direct+ --packages-select rcl_lifecycle --ctest-args -R test_rcl_lifecycle
...
The following tests passed:
        test_rcl_lifecycle

100% tests passed, 0 tests failed out of 1

Label Time Summary:
gtest    =   1.81 sec*proc (1 test)

Total Test time (real) =   1.81 sec
Finished <<< rcl_lifecycle [2.48s]

Summary: 1 package finished [3.67s]
```

I am not sure if this is the correct thing to do... maybe something wrong with rmw_zenoh instead?